`cargo install durt` fails for 1.0.0 (windows)

`cargo install durt` now fails for 1.0.0. Strangely, `cargo build --release` works just fine from the repository, although, again, `cargo install --path .` fails.

There are a couple of dependency differences.

I've included the build output here below.

```shell
C:>cargo install durt
    Updating crates.io index
  Installing durt v1.0.0
   Compiling winapi v0.3.9
   Compiling version_check v0.9.2
   Compiling proc-macro2 v1.0.24
   Compiling unicode-xid v0.2.1
   Compiling syn v1.0.44
   Compiling autocfg v1.0.1
   Compiling bitflags v1.2.1
   Compiling unicode-segmentation v1.6.0
   Compiling unicode-width v0.1.8
   Compiling hashbrown v0.9.1
   Compiling strsim v0.10.0
   Compiling lazy_static v1.4.0
   Compiling vec_map v0.8.2
   Compiling os_str_bytes v2.3.2
   Compiling glob v0.3.0
   Compiling number_prefix v0.4.0
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling indexmap v1.6.0
   Compiling heck v0.3.1
   Compiling textwrap v0.12.1
   Compiling tabular v0.1.4
   Compiling wild v2.0.4
   Compiling quote v1.0.7
   Compiling winapi-util v0.1.5
   Compiling atty v0.2.14
   Compiling ansi_term v0.12.1
   Compiling termcolor v1.1.0
   Compiling same-file v1.0.6
   Compiling walkdir v2.3.1
   Compiling clap_derive v3.0.0-beta.2
   Compiling clap v3.0.0-beta.2
   Compiling durt v1.0.0
error[E0308]: mismatched types
 --> C:\Users\Roy\.cargo\registry\src\github.com-1285ae84e5963aae\durt-1.0.0\src\cli.rs:8:10
  |
8 | #[derive(Clap)]
  |          ^^^^ expected `char`, found `&str`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: failed to compile `durt v1.0.0`, intermediate artifacts can be found at `C:\Users\Roy\AppData\Local\Temp\cargo-installHAg4lC`

Caused by:
  could not compile `durt`.

To learn more, run the command again with --verbose.
```

```shell
C:>cd durt\repo
C:>git describe --tags --long
v1.0.0-0-gd6e020e

C:\Users\Roy\OneDrive\Projects\rust\durt\repo>cargo build --release
   Compiling proc-macro2 v1.0.18
   Compiling winapi v0.3.9
   Compiling version_check v0.9.2
   Compiling unicode-xid v0.2.1
   Compiling syn v1.0.33
   Compiling autocfg v1.0.0
   Compiling unicode-width v0.1.8
   Compiling bitflags v1.2.1
   Compiling unicode-segmentation v1.6.0
   Compiling strsim v0.10.0
   Compiling os_str_bytes v2.3.1
   Compiling glob v0.3.0
   Compiling lazy_static v1.4.0
   Compiling vec_map v0.8.2
   Compiling number_prefix v0.4.0
   Compiling proc-macro-error-attr v0.4.12
   Compiling proc-macro-error v0.4.12
   Compiling indexmap v1.4.0
   Compiling textwrap v0.11.0
   Compiling tabular v0.1.4
   Compiling heck v0.3.1
   Compiling wild v2.0.4
   Compiling quote v1.0.7
   Compiling winapi-util v0.1.5
   Compiling atty v0.2.14
   Compiling ansi_term v0.12.1
   Compiling termcolor v1.1.0
   Compiling same-file v1.0.6
   Compiling syn-mid v0.5.0
   Compiling walkdir v2.3.1
   Compiling clap_derive v3.0.0-beta.1
   Compiling clap v3.0.0-beta.1
   Compiling durt v1.0.0 (C:\Users\Roy\OneDrive\Projects\rust\durt\repo)
    Finished release [optimized] target(s) in 35.11s

C:\Users\Roy\OneDrive\Projects\rust\durt\repo>cargo install --path
error: The argument '--path <PATH>' requires a value but none was supplied

USAGE:
    cargo.exe install --path <PATH>

For more information try --help

C:>cargo install --path .
  Installing durt v1.0.0 (C:\Users\Roy\OneDrive\Projects\rust\durt\repo)
    Updating crates.io index
   Compiling proc-macro2 v1.0.24
   Compiling syn v1.0.44
   Compiling autocfg v1.0.1
   Compiling hashbrown v0.9.1
   Compiling os_str_bytes v2.3.2
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling textwrap v0.12.1
   Compiling indexmap v1.6.0
   Compiling quote v1.0.7
   Compiling clap_derive v3.0.0-beta.2
   Compiling clap v3.0.0-beta.2
   Compiling durt v1.0.0 (C:\Users\Roy\OneDrive\Projects\rust\durt\repo)
error[E0308]: mismatched types
 --> src\cli.rs:8:10
  |
8 | #[derive(Clap)]
  |          ^^^^ expected `char`, found `&str`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: failed to compile `durt v1.0.0 (C:\Users\Roy\OneDrive\Projects\rust\durt\repo)`, intermediate artifacts can be found at `C:\Users\Roy\OneDrive\Projects\rust\durt\repo\target`

Caused by:
  could not compile `durt`.

To learn more, run the command again with --verbose.
``

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,4 +18,4 @@ ansi_term = "0.12.1"
 tabular = "0.1.4"
 wild = "2.0.4"
 walkdir = "2.3.1"
-clap = "3.0.0-beta.1"
+clap = { version="3.0.0-beta.2" }

--- a/src/cli.rs
+++ b/src/cli.rs
@@ -12,11 +12,11 @@ pub struct Cli {
     /// Paths to files or directories. Use wildcards for recursion
     pub paths: Vec<PathBuf>,
 
-    #[clap(short = "b", long = "binary")]
+    #[clap(short = 'b', long = "binary")]
     /// Use binary prefixes (Ki, Mi, Gi, etc.) instead of decimal
     pub use_binary_prefixes: bool,
 
-    #[clap(short = "P", long = "percentage")]
+    #[clap(short = 'P', long = "percentage")]
     /// Show each entry's percentage relative to the total
     pub show_percentages: bool,
 
@@ -24,7 +24,7 @@ pub struct Cli {
     /// Omit entries with size less than this
     pub minimum_percentage: Option<f64>,
 
-    #[clap(short = "t", long = "total")]
+    #[clap(short = 't', long = "total")]
     /// Print the sum of all sizes at the end
     pub show_total: bool,
 
@@ -32,7 +32,7 @@ pub struct Cli {
     /// Print entries in ascending order of size
     pub sort: bool,
 
-    #[clap(short = "p", long = "by-path")]
+    #[clap(short = 'p', long = "by-path")]
     /// Sort by path instead of by size
     pub sort_by_path: bool,
 
@@ -41,7 +41,7 @@ pub struct Cli {
     pub reverse_order: bool,
 
     #[cfg(unix)]
-    #[clap(short = "f", long)]
+    #[clap(short = 'f', long)]
     /// Ignore entries from filesystems different from that of the first path passed
     pub same_fs: bool,
 }

